sdk/openai/openai/test/public/openai.spec.ts

```diff
@@ -217,7 +217,26 @@ describe("OpenAI", function () {
                     .getChatCompletions(deploymentName, weatherMessages, {
                       functions: [getCurrentWeather],
                     })
-                    .then((c) => assertChatCompletions(c, { functions: true }))
+                    .then((result) => {
```

Comment:
nitpick: would you think that using async functions for lengthy promise chains make them easier to read?